### PR TITLE
Bump to 4.12.0.

### PR DESCRIPTION
**Pull request by jezdez**
Release: https://github.com/conda/conda/releases/tag/4.12.0

### CODE REVIEW

**Comment by skupr-anaconda** [comment]
Hmm, an error is:
`OSError: [Errno 28] No space left on device`

**Comment by LtDan33** [comment]
@skupr-anaconda Did you do anything to get that memory error fixed or just reran the tests?

File: recipe/meta.yaml [hunk 1]
```diff
@@ -7,7 +7,7 @@ package:
 source:
   fn: conda-{{ version }}.tar.gz
   url: https://github.com/conda/conda/archive/{{ version }}.tar.gz
-  sha256: 1843355ccb93afb05f2a307e1fc37403fb9976da799236eebc3adee1c716c5fc
+  sha256: c6607fee920aae5d96528669fc9e61b9beff6cf8c634d1d85bc6f409d5146b44
 
 build:
   number: 0
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:13]: We should add `skip: True  # [py<36]`, https://github.com/conda/conda/blob/fc81d4217f6c1804e983d1698f8b7cc5bc01874c/setup.py#L85

Comment (skupr-anaconda) [inline recipe/meta.yaml:13]: There should be entry_points:
```
  entry_points:
    - conda=conda.cli.main_pip:main
```
https://github.com/conda/conda/blob/fc81d4217f6c1804e983d1698f8b7cc5bc01874c/setup.py#L79

Comment (jezdez) [reply recipe/meta.yaml:13]: Please don't add entrypoints for the conda python packaging integration, for historical reasons ignore the setup.py for conda for now, it's defunct. The `conda` command line script is an alias provided by its shell integration.

Comment (jezdez) [reply recipe/meta.yaml:13]: Good catch!

**Review by chenghlee — APPROVED** [review]

File: recipe/meta.yaml (hunk 1 above)
Comment (ScottBailey) [reply recipe/meta.yaml:14]: Updated with @skupr-anaconda's comment.

**Pull request merged**